Assets/HoloToolkit/Input/Scripts/GameControllerManipulator.cs

```diff
@@ -0,0 +1,145 @@
+﻿using System;
+using UnityEngine;
+using System.Collections;
+using HoloToolkit.Unity;
+
+namespace HoloToolkit.Unity
+{
+    /// <summary>
+    /// Allows the user to place and rotate GameObjects using a game controller.
+    /// </summary>
+    /// <remarks>Make sure to enable the HumanInterfaceDevice capability before using.</remarks>
```

Comment:
I couldn't find any way to verify this capability :( Unity also seems to swallow any exception that may be raised.